mlflow/java/client/src/main/java/org/mlflow/MlflowMetric.java

```diff
@@ -0,0 +1,10 @@
+package org.mlflow;
+
+public class MlflowMetric {
```

Comment:
Any reason these have MLflow in the name despite being part of the MLflow package? It seems a bit weird. Also, are there equivalent classes in Python, and what are those named?

Reply:
Also, we may want to put these in the tracking package too or in a data package if that's what we have in Python.